utils/defaulter.go

```diff
@@ -44,13 +45,20 @@ func getKongDefaulter(opts DefaulterOpts) (*Defaulter, error) {
 	if err := d.populateDefaultsFromInput(opts.KongDefaults); err != nil {
 		return nil, err
 	}
+
+	// TODO check if it's OK to do the validation only for the user input
```

Comment:
What do you mean with this?

Reply:
not applicable anymore with the refactor